config/routes.rb

```diff
@@ -198,7 +198,7 @@
       resources :partnerships, only: :index
       resource :programme, only: %i[edit], controller: "choose_programme"
 
-      resources :participants, only: %i[index show], constraints: ->(_request) { FeatureFlag.active?(:induction_tutor_manage_participants) } do
+      resources(:participants, only: %i[index show]) do
```

Comment:
Can we use spaces instead of brackets (like in other places in this file?)

Reply:
Yup, I was trying to preserve the constraint which forced me to split the call into multiple lines, hence brackets. They need to go though.